src/block/CeilingEdgesHangingSign.php

```diff
@@ -33,6 +35,7 @@
 
 final class CeilingEdgesHangingSign extends BaseSign implements HorizontalFacing{
 	use HorizontalFacingTrait;
+	use StaticSupportTrait;
```

Comment:
I don't think we can use `StaticSupportTrait` for this. Since the support conditions depend on the current state of the block.

`StaticSupportTrait` logic is supposed to be independent of the block's state.